.pre-commit-config.yaml

```
@@ -95,3 +95,8 @@ repos:
       - id: prettier
         args: [--prose-wrap=always, --print-width=88]
         exclude: tests(/.*)*/data
+  - repo: https://github.com/DanielNoord/pydocstringformatter
+    rev: v0.1.0
+    hooks:
+      - id: pydocstringformatter
+        exclude: tests/functional/|tests/input|tests(/.*)*/data|tests/regrtest_data/|tests/data/
```

```suggestion
        exclude: *fixtures
```
Or is that different than the fixture we have otherwise ?

---

I didn't know about this!